gekitchensdk/erd/values/laundry/erd_cycle.py

```diff
@@ -0,0 +1,59 @@
+import enum
```

Comment:
This file looks duplicative of erd_laundry_cycle

Reply:
you're correct, this was a duplicate, i've deleted it.